### ISSUE TITLE
Screen Timeout issue

### ISSUE THREAD

**Issue by gitisara** [bigbluebutton/bigbluebutton#9572]
**Describe the bug**
Screen gets off during the live meeting , when not on full screen.

**To Reproduce**
1. Go to Meeting
2. Join Meeting
3. Let it play
4. Screen will not lock to AWAKE
5.Screen will get turn off.

**Expected behavior**
Screen should remain awake

**Actual behavior**
Screen should be awake when meeting is going on.

**Screenshots**

**Smartphone (please complete the following information):**
 - Device: Android
 - Browser : Chrome

**Comment by antobinary** [comment]
Hi @gitisara ,

I see it as an advantage that the screen can be off and I can still participate in a long meeting (listen only perhaps) without battery drain. You still have the option to make the client full screen, and looks like then it keeps the screen on.

Let's hear more opinions on the subject

**Comment by gitisara** [comment]
Yes , but the same can be achieved by just pressing Power Button Once.

It becomes Pain in real life classes , As screen Getting turn off automatically during Polling.

They are reading Questions on Presentation screen , thinking about the answer....and screen gets off..
That is very annoying.

And since there is no Picture In Picture mode in fullscreen.
if someone wishes to see teacher and the presentation same time..

Same pain again.

**Comment by antobinary** [comment]
I see what you're saying. Meanwhile you can tweak a setting on your device (location may vary) Settings -> Display -> Sleep -> [here I can set the screen to timeout after 30 mins since last touch, and if you increase-decrease volume every 30 mins you'll be good]

Also, Polling triggers a "beep" sound when it's initiated by the presenter...

In the past I've seen some setting where if there is video playing on the screen, the screen would not be timing out but I'm not finding info about it now.
Just offering interim solutions.

**Comment by vj-abishek** [comment]
It is still in progress? It will be useful when the screen lock is enabled when entered in fullscreen, in the presentation. My college uses just a presentation to teach. They are not using video stream, video to teach. It is very annoying when the screen goes off every 30s. 

It will be good if the wake lock is enabled when entered into fullscreen mode, in presentation and disabling it when exiting fullscreen mode. 

I think wake lock API is supported in 66% of the browsers. Then why don't you try it? 
Or [try this package](https://www.npmjs.com/package/nosleep.js) for android and ios

Please release it soon.

**Comment by biszinc** [comment]
Adding our use case to the story.  we operate a platform to allow junior school teacher to provider supplementary class to < grade 6 student.

Student using ipad/iphone continuously complain about the screen go dark problem.   Considering the attendee are very young the screen look issue is disrupting the class very badly.

Would appreciate if this feature can be implemented, if general support is a concern, maybe implement this as an optional feature admin can turn on/off when they create the room, so administrator can decide and be alerted of the limit the feature offer?

**Comment by wemoeller** [comment]
is there data on current behavior?  can you give info on  android/ios versions, browser-versions and bbb-versions?

**Comment by FreeScholar** [comment]
Found this API - https://caniuse.com/?search=wake%20lock

**Comment by FreeScholar** [comment]
> is there data on current behavior? can you give info on android/ios versions, browser-versions and bbb-versions?

version 2.4 will do best to get phone versions from users that had issues! Thanks to all who are looking at this!

**Comment by shabeepk** [comment]
Has there been any progress on this issue?
